### ISSUE TITLE
JSON Output

### ISSUE THREAD

**Issue by ohansck** [ts-oas/ts-oas#3]
The final output does not come out as a json, but as a javascript object.

**Comment by alireza-mf** [comment]
Yes, there was a mismatch in the doc. Now I clarified in [This commit](https://github.com/ts-oas/ts-oas/commit/7a156d5a9d505430c298f0bf3fa10966cafa981a).
Thanks.
